test/helpers/envtest.go

```diff
@@ -31,19 +31,19 @@ import (
 	"k8s.io/klog/klogr"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
 	bootstrapv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha3"
-	"sigs.k8s.io/cluster-api/cmd/clusterctl/log"
 	"sigs.k8s.io/cluster-api/controllers/external"
 	expv1 "sigs.k8s.io/cluster-api/exp/api/v1alpha3"
 	"sigs.k8s.io/cluster-api/util"
 	"sigs.k8s.io/cluster-api/util/kubeconfig"
+	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
 func init() {
 	klog.InitFlags(nil)
-	log.SetLogger(klogr.New())
+	ctrl.SetLogger(klogr.New())
```

Comment:
Do we not want to include `clusterctl/log.SetLogger`?
This is one place where `clusterctl/log` is still being used.
https://github.com/kubernetes-sigs/cluster-api/blob/80653d174c6f5bb5ea76fbafbe2f0c50cc1185dd/controllers/suite_test.go#L59-L66

Reply:
Ah, I didn't notice it being used with a quick check

Reply:
Should this clusterctl package even be public?

Reply:
Since there is a different instance of `Log` from the `clusterctl/log` package.

https://github.com/kubernetes-sigs/cluster-api/blob/80653d174c6f5bb5ea76fbafbe2f0c50cc1185dd/cmd/clusterctl/log/log.go#L24-L32

Reply:
Fixed it up, let's follow up to understand why we need to have this clusterctl log package public, seems it could be internal instead?

Reply:
Yeah..I think it's valid to keep it internal. Other than the clusterctl packages, it is being used in the `controllers/suite_test.go` so maybe we can refactor it out.

Reply:
> Fixed it up, let's follow up to understand why we need to have this clusterctl log package public, seems it could be internal instead?
@vincepri Did you mean to add `log.SetLogger(klogr.New())` back in? If so I'm not seeing the change.

Reply:
Yeah, we shouldn't be using clusterctl logger anywhere in here